### PR TITLE
release-notes: updates for 20230905 releases

### PR DESCRIPTION
**Pull request by dustymabe**

### CODE REVIEW

**Review by jlebon — DISMISSED** [review]
Trivial LGTM

File: release-notes/stable.yml [hunk 1]
```diff
@@ -1,4 +1,8 @@
 releases:
+  38.20230902.1.0:
```
Comment (marmijo) [inline release-notes/stable.yml:2]: This should be `38.20230819.3.0`

**Review by marmijo — COMMENTED** [review]
I left one comment. Otherwise, LGTM

File: release-notes/stable.yml (hunk 1 above)
Comment (dustymabe) [reply release-notes/stable.yml]: Thanks for catching this! Fixed

**Review by marmijo — APPROVED** [review]
Thanks, LGTM!

**Review by jlebon — APPROVED** [review]

**Pull request merged**